### PR TITLE
fixed seeds for likes

### PR DESCRIPTION
**Pull request by Zoneam**

### CODE REVIEW

**Review by melinamboedecker — APPROVED** [review]

File: server/seeders/seed.js [hunk 1]
```diff
@@ -8,17 +8,17 @@ const userSeed = require("./userSeed.json");
 db.once("open", async () => {
   try {
     // remove from database
-    await Category.deleteMany({});
-    await Label.deleteMany({});
+    //await Category.deleteMany({});
```
Comment (vasylynash) [inline server/seeders/seed.js]: Delete

File: server/seeders/seed.js [hunk 2]
```diff
@@ -8,17 +8,17 @@ const userSeed = require("./userSeed.json");
...
+    //await Label.deleteMany({});
```
Comment (vasylynash) [inline server/seeders/seed.js]: Delete

File: server/seeders/seed.js [hunk 3]
```diff
@@ -8,17 +8,17 @@ const userSeed = require("./userSeed.json");
...
     await Post.deleteMany({});
     await User.deleteMany({});
 
     // const categories = await Category.insertMany(categorySeed);
```
Comment (vasylynash) [inline server/seeders/seed.js]: Delete

File: server/seeders/seed.js [hunk 4]
```diff
@@ -8,17 +8,17 @@ const userSeed = require("./userSeed.json");
...
-    const labels = await Label.insertMany(labelSeed);
+    //const labels = await Label.insertMany(labelSeed);
```
Comment (vasylynash) [inline server/seeders/seed.js]: Delete

File: server/seeders/seed.js [hunk 5]
```diff
@@ -8,17 +8,17 @@ const userSeed = require("./userSeed.json");
...
     const posts = await Post.insertMany(postSeed);
     const users = await User.insertMany(userSeed);
 
-    // //Post references
+    //Post references
```
Comment (vasylynash) [inline server/seeders/seed.js]: Delete

**Review by vasylynash — CHANGES_REQUESTED** [review]

**Review by vasylynash — APPROVED** [review]

**Pull request merged**